VideoWeb/VideoWeb/ClientApp/src/app/services/user-media.service.ts

```diff
@@ -88,6 +91,14 @@ export class UserMediaService {
         return this.getCachedDeviceIfStillConnected(this.preferredMicCache);
     }
 
+    getShowDialogChooseDevice() {
```

Comment:
same as above, this service class should not track whether or not a dialog needs to be shown or tracking the user's visits for the WR

Reply:
moved to wr base class